docs/installation-configuration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Aquasec Setup
+meta_desc: Information on how to install the Aquasec provider.
+layout: installation
+---
+
+## Installation
+
+The Pulumi Aquasec provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumiserve/pulumi-aquasec`](https://www.npmjs.com/package/@pulumiverse/aquasec)
```

Comment:
2 typo's in the package name:
- it is `pulumiverse`, not `pulumiserve` 😉 
- it is `aquasec` rather than `pulumi-aquasec`

per this line: https://github.com/pulumiverse/pulumi-aquasec/blob/main/provider/resources.go#L151

Reply:
Good catch, Ringo! 👍